src/olympia/amo/tests/test_middleware.py

```diff
@@ -116,8 +116,8 @@ class TestNoDjangoDebugToolbar(TestCase):
     def test_no_django_debug_toolbar(self):
         with self.settings(DEBUG=False):
             res = self.client.get(reverse('home'), follow=True)
-            assert 'djDebug' not in res.content
-            assert 'debug_toolbar' not in res.content
+            assert b'djDebug' not in res.content
+            assert b'debug_toolbar' not in res.content
```

Comment:
Let's use `force_text(res.content)` everywhere maybe?

Reply:
Or `u'djDebug' not in res.content.decode('utf-8')` but we should try to find one style to do this in all our tests primarily

Reply:
fwiw, we've used the `b'foo'` fix _a lot_ in our other test changes (i.e. I'm not saying we shouldn't do this - and I made the same suggestion in another pr - but we're already far down the road of testing for bytestrings in raw `.content`)

Reply:
Yeah, thinking about it, I don't think it matters much anyway. Any way works in regards to testing so :man_shrugging: